workers_routes.go

```diff
@@ -0,0 +1,161 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+var ErrMissingWorkerRouteID = errors.New("missing required route ID")
+
+type ListWorkerRoutes struct{}
+
+type CreateWorkerRouteParams struct {
+	Pattern string `json:"pattern"`
+	Script  string `json:"script,omitempty"`
+}
+
+type ListWorkerRoutesParams struct{}
+
+type UpdateWorkerRouteParams struct {
+	ID      string `json:"id,omitempty"`
+	Pattern string `json:"pattern"`
+	Script  string `json:"script,omitempty"`
+}
+
+// CreateWorkerRoute creates worker route for a script.
+//
+// API reference: https://api.cloudflare.com/#worker-routes-create-route
+func (api *API) CreateWorkerRoute(ctx context.Context, rc *ResourceContainer, params CreateWorkerRouteParams) (WorkerRouteResponse, error) {
+	if rc.Level != ZoneRouteLevel {
+		return WorkerRouteResponse{}, fmt.Errorf(errInvalidResourceContainerAccess, ZoneRouteLevel)
+	}
+
+	if rc.Identifier == "" {
+		return WorkerRouteResponse{}, ErrMissingIdentifier
+	}
+
+	uri := fmt.Sprintf("/zones/%s/workers/routes", rc.Identifier)
```

Comment:
worker routes still use the `/workers/routes` endpoint instead of the services route as that isn't documented. once we have that documented, we can swap it over.